tools/config.sh

```diff
@@ -257,17 +257,6 @@ defvar R_CONTAINER       "rrr"
 defvar -xX R_URL         "$(_main_url "$R_CONTAINER")"
 defvar -dX R_PACKAGE     "mmlspark-<{MML_VERSION}>.zip"
 
-# E2E test cluster information
```

Comment:
Instead of removing all of the `E2E` stuff, it should be the name used for the new stuff since these are the new end to end tests.

Reply:
im going to make e2e refer to any tests that occur after publishing aka local and adb notebooks

Reply:
My comment was about `adb` in the name: it's an implementation detail that is unrelated to *having* E2E tests.  Just like there were no `hdinsight` names previously at the abstract level.